frontend/src/views/AdminPages/BranchesPage/BranchComponent/BranchComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+// core components
+import Button from "components/CustomButtons/Button.js";
+
+export default function BranchComponent({branch, onEditClick, onDeleteClick}) {
+    // const {branch} = props; 
+    return (
+        <tr>
+            <td>{branch.branchCity}</td>
+            {window.innerWidth >= 768 ?
```

Comment:
Prefer css to handle breakpoints for responsiveness. This will only apply when rendered. If the user changes the size of their window after, they wont get the intended view.

Reply:
Fixed this.